frontend/feature/event/src/main/java/com/intersoft/event/EventsViewModel.kt

```diff
@@ -12,8 +12,14 @@ class EventsViewModel : ViewModel(){
 
     private val _events: MutableLiveData<List<IIterableObject>> = MutableLiveData(null)
     private val _error : MutableLiveData<String> = MutableLiveData("")
+    private val _canJoin: MutableLiveData<Boolean> = MutableLiveData(false)
+
     val events: LiveData<List<IIterableObject>> = _events
     val error: LiveData<String> = _error
+    val canJoin: LiveData<Boolean> = _canJoin
+
+    val CAN_JOIN = "Successfully joined the Event."
```

Comment:
Could have these strings in an xml file (res folder) which could then be accessed through the R class